kube-enforcer/templates/kube-enforcer-deployment.yaml

```diff
@@ -81,7 +81,8 @@ spec:
           {{- include "extraSecretEnvironmentVars" .Values | nindent 10 }}
           ports:
           {{- if not .Values.kubeEnforcerAdvance.enable }}
-            - containerPort: 8443
+            - containerPort: {{ .Values.ports.tlsContainerPort }}
+              name: https
           {{- else }}
             - containerPort: 8449
```

Comment:
what about taking these defaults out to the values.yaml?

Reply:
I exactly suggested move these values ​​to values.yaml =)

Reply:
are we talking about ports 8449 and 8442 in the else statement?

Reply:
maybe same change should be made also in else statement (if kubeEnforcerAdvance "enable" true).
current PR focus on "regular" KE mode.

and i don't know, why is used port 8449 insted of smth else in KE Advence mode

Reply:
Hello @ultramaxim 

From the below output, we can see the "kubeenforceradvance.enable" section updated to "containerPort: {{ .Values.ports.tlsContainerPort }}".  
So, We need to enable the kube enforcer advance section in the values.yaml highlighted below to set the port values to be changed as per requirement.

Please let us know if you agree with this.

![image](https://github.com/user-attachments/assets/bd6bae0a-656f-47d7-a04a-2c7b2e245980)
![image](https://github.com/user-attachments/assets/db269202-acdf-4552-b217-d8b05ff90f97)


Regards,
Srikanth Reddy

Reply:
hello,  but the if statement (line 83) contains boolean operation NOT (if **NOT** .values.kubeEnforcerAdvance.enable)

thats why my settings will not in kubeEnforcerAdvance section in values.yaml